app/views/govuk_publishing_components/components/_breadcrumbs.html.erb

```diff
@@ -2,14 +2,16 @@
   breadcrumbs ||= []
   inverse ||= false
   invert_class = inverse ? "gem-c-breadcrumbs--inverse" : ""
+  collapse_on_mobile ||= true unless collapse_on_mobile == false
+  collapse_on_mobile_class = collapse_on_mobile ? "gem-c-breadcrumbs--collapse-on-mobile" : ""
   breadcrumb_presenter = GovukPublishingComponents::Presenters::Breadcrumbs.new(breadcrumbs)
 %>
 
 <script type="application/ld+json">
   <%= raw breadcrumb_presenter.structured_data.to_json %>
 </script>
 
-<div class="gem-c-breadcrumbs govuk-breadcrumbs <%= invert_class %>" data-module="track-click">
+<div class="gem-c-breadcrumbs govuk-breadcrumbs <%= invert_class %> <%= collapse_on_mobile_class %>" data-module="track-click">
```

Comment:
Might be more readable to stack the classes like in other components: https://github.com/alphagov/govuk_publishing_components/blob/master/app/views/govuk_publishing_components/components/_document_list.html.erb#L4